webapp-django/crashstats/crashstats/utils.py

```diff
@@ -14,7 +14,7 @@
 from django import http
 from django.conf import settings
 
-from . import models
+import models
```

Comment:
This isn't valid Python 3. It needs to either be:

```
from crashstats.crashstats import models
```

or
 
```
from . import models
````

Out of curiosity, what prompted you to change it?